next-frontend/src/lib/i18n/get18n.ts

```diff
@@ -0,0 +1,19 @@
+import i18next from "./i18n";
+// import { headerName } from "./settings";
+// import { headers } from "next/headers";
+
+export async function getT(options?: { keyPrefix?: string }) {
+  // const headerList = await headers();
+  const lng = "en";
+  if (lng && i18next.resolvedLanguage !== lng) {
+    await i18next.changeLanguage(lng);
+  }
```

Comment:
Why do you have to check this during every single method call? I think for someone who just casually browses the website in French, this will trigger a `changeLanguage` for every single translation string that is rendered, no?